exercises/modal.js

```diff
@@ -3,25 +3,30 @@
 class Modal{
 	//constructor takes in 3 arguments: the modal shadow dom element, the modal body dom element, and the modal message dom element
 	//Modal will have a property called onClose that will eventually be set to a callback set by the outside
-	constructor( ){
-
+	constructor( modalShadow, modalBody, modalMessage ){
+		this.modalShadow = $(modalShadow);
+		this.modalBody = $(modalBody);
+		this.modalMessage = $(modalMessage);
+		this.onClose = null;
 	}
 	//takes in no arguments.
 	//returns nothing
 	//shows the given modal shadow and the given modal body
 	show(){
-		
+		this.modalShadow.css('display', 'unset');
```

Comment:
hrm, I've never tried that before

Reply:
Technically if you hide and show in jquery, it returns the value to what it was last time.